feat: Add ListMetadata + Read + Write functionality to Attio connector (#1046)
amp-labs/connectors: providers/attio/README.md

# Atio connector

## Supported Objects 

> Below is an exhaustive list of the supported Objects in the Attio deep connector with their endpoint resources(ObjectName).

Reviewer: ```suggestion
Below is an exhaustive list of objects & methods supported on the objects -
```

Attio API version : v2

> | Object | Resource | Method

Reviewer: Write seems to be missing as of now?

Reviewer 2: I think objectNames.go is an exact representation of this README. I would remove README, maybe would add comments to that file instead. There is always an incentive to keep go code up to date unlike `.md`, `.txt` type of files.
